l10n_us_account_profile/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2017 Ursa Information Systems <http://www.ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "US Accounting",
+    "summary": "Additional features to manage US accounting in Odoo",
+    "version": "11.0.1.0.0",
+    "category": "Accounting",
+    "website": "http://www.ursainfosystems.com",
```

Comment:
@Sudarshan1607 Please change it to 'website': 'https://github.com/OCA/l10n-usa',